Update limit for trace filter (#1502)
hyperledger/besu-docs: docs/public-networks/reference/api/index.md

### `trace_filter`

> Returns traces matching the specified filter. The default block range limit for `trace_filter` is 1000. You can adjust this limit using the [`--rpc-max-trace-filter-range`](../cli/options.md#rpc-max-trace-filter-range) option. 

Reviewer: ```suggestion
Returns traces matching the specified filter. The maximum number of blocks you can supply to `trace_filter` is 1000 by default. You can adjust this limit using the [`--rpc-max-trace-filter-range`](../cli/options.md#rpc-max-trace-filter-range) option. 
```

Your node must be an archive node (that is, synchronized without pruning or fast sync) or the requested block must be within the number of [blocks retained](../cli/options.md#pruning-blocks-retained) with [pruning enabled](../cli/options.md#pruning-enabled) (by default, 1024).

> 
> 

Reviewer: ```suggestion
```

Why these added line breaks?

</TabItem>
</Tabs>

> The default block range limit for `trace_filter` is 1000. You can adjust this limit using the `--rpc-max-trace-filter-range` option. 

Reviewer: I'm not sure this needs to be mentioned here, but if it does, please move it up into the description for this method ("Returns traces matching the specified filter.") instead of after the example.
